integration_tests/mockApis/orchestration.ts

```diff
@@ -38,6 +38,37 @@ export default {
     })
   },
 
+  stubCancelVisit: ({
+    applicationReference,
```

Comment:
```suggestion
    reference,
```
Suggest renaming this to avoid confusion between **application** and **visit** references.